src/libprojectM/PCM.cpp

```diff
@@ -125,7 +125,7 @@ void PCM::addPCMfloat(const float* PCMdata, size_t samples)
         size_t j=(i+start)%maxsamples;
         a=pcmL[j] = PCMdata[i];
         pcmR[j] = PCMdata[i];
-        sum += fabs(a);
+        sum += std::abs(a);
```

Comment:
Suggest placing a `using std::abs;` at the top of the implementation file to make the calls a bit less verbose (same for `std::max`).

Reply:
I'm not a fan of file-wide using declarations when it comes to libraries, but if you think it'd be more readable maybe they can be put locally? Always `std::abs` would be consistent, if we have exceptions to that I think we should indicate that as locally as possible. Personally my brain just filters out the `std::`, like lipsers filter out all the parentheses when they code :^)

Reply:
Can just keep it that way. Would be more of an issue with a class in a nested namespace, like the Poco library often has, but in this case `using namespace` is a good solution if these classes are heavily used.